filterizePy/test/test_mirror.py

```diff
@@ -7,47 +7,64 @@
 # This script tests the function from mirror.py.
 
 # This script tests the mirror function of the filterizePy package.
-
-from skimage import io
-import pytest
+import sys
+sys.path.append('../../')
+from  skimage import io
 import numpy as np
-from filterize import mirror
+import pytest
+from filterizePy.mirror import mirror
 
-input_img = io.imread('../img/input_image.png')
-mirror(input_img)
-output_img = io.imread('../img/mirror_img.png')
+# class test_mirror:
```

Comment:
Hi Jack,
I guess you could remove the commented code which is not being used. Rest all looks good!